.github/workflows/node.js.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
qs, what is this for? when we merge PRs it will run both `push` tests & `pull_request` tests? or just `push` tests?

Reply:
There will be unit tests triggered in push and pull request event.
I think I'll remove the `on pull request`, since it is also what we had in the previous config.